### PR TITLE
Remove jgoodies look & feel themes

### LINKED ISSUE #151: Playlist Editor Tabs become unreadable when changing Look & Feel
After changing Look & Feel (in options):

image

### PR DESCRIPTION
**Pull request by Borewit**
Changes:
- Removes the [JGoodies Looks](https://www.jgoodies.com/freeware/libraries/looks/) look & feel themes, as they seem to introduce buggy behaviour.
- Allow the ["Nimbus"  look & feel theme](https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/nimbus.html) to be selected.
- Some code optimization in `AppOptionsDialog.java`
- Set default look & feel to ["Nimbus"  look & feel theme](https://docs.oracle.com/javase/tutorial/uiswing/lookandfeel/nimbus.html), was the Operating System default theme

Drop dependencies:
1.  `com.jgoodies:jgoodies-common:1.8.1`
2.  `com.jgoodies:jgoodies-looks:2.7.0`

Resolves: #151

### CODE REVIEW

**Comment by Borewit** [comment]
Another _less is more_ PR, @touwys, what do you think?

Build: [listFix_2.6.0-PR152-49.exe](https://workupload.com/file/qcEvMhqVQM2)

**Comment by touwys** [comment]
> Another _less is more_ PR, @touwys, what do you think?

@Borewit, I can try this build out tomorrow. We're scheduled for another blackout in 45 minutes. I'm really keen to see what you've done to the "look & feel" of the app.

**Comment by touwys** [comment]
@Borewit, Curiousness got the best of me, and I ran a quick test before shutting down:

**NIMBUS** is, for me, the **best by far** — a keeper, even if the only one, and set as default. It is followed by Metal, Windows, Windows Classic. CDE/Motif, falls short on several accounts.

Nice work! 👍🏻

**Comment by Borewit** [comment]
Nimbus was explicitly excluded, so I just get rid of that.

I have changed the default Look & Feel to Nimbus via commit 53baf16173807317f35ca22c1b3563fee0239d06

**Comment by touwys** [comment]
@Borewit:

**Good going**:tm: with your release of [listFix() v2.7.0](https://github.com/Borewit/listFix/releases). I've already got v2.7.0 installed. 

As I continue to test it intermittently, should I inclusively post comments, observations, or issues — all under ["Issues"](https://github.com/Borewit/listFix/issues)? **Proposal**: _It could be fruitful to open a separate subject that takes care of the general discussion around current listFix() releases_.

Have you finished picking all the "low-hanging fruit" now, or are there more to come? If you would, please see whether you can fit into your plans, _[this particular issue](https://github.com/Borewit/listFix/issues/116#issue-1600016422)_: It's a bother to identify a matched file — especially so if it's to be found quickly somewhere among numerous other entries — once its location is established.

**Comment by midmarch** [comment]
> As I continue to test it intermittently, should I inclusively post comments, observations, or issues — all under ["Issues"](https://github.com/Borewit/listFix/issues)? Proposal: It could be fruitful to open a separate subject that takes care of the general discussion around current listFix() releases.

Ideally issues are requests for change, bug requests, feature request, improvement. Clearly scoped change. Discussions in issues should be primarly about that issue. Same goes for the PR, which is about a change, which could resolve 1 or more issues. 
 
I have actually never used, but we could use [discussions](https://github.com/Borewit/listFix/discussions) for dicussions beyond the scope of issue's and PR's. But that is actually what it is designed for. For other projects I used Discord, which offers a bit more privacy.

> Have you finished picking all the "low-hanging fruit" now, or are there more to come? If you would, please see whether you can fit into your plans, https://github.com/Borewit/listFix/issues/116#issue-1600016422: It's a bother to identify a matched file — especially so if it's to be found quickly somewhere among numerous other entries — once its location is established.

It is not the end of the world, but that comment does not belong here, it belongs in #116. If we discuss it here, it will be hard to follow for others, we create confusing links from this issue to that that issue etc
